cellprofiler/preferences.py

```diff
@@ -206,7 +205,7 @@ def config_read(key):
         value = None
     if value is not None:
         try:
-            value = utf16decode(value)
+            value = cellprofiler.utilities.utf16encode.utf16decode(value)
```

Comment:
Could this be replaced with the corresponding Python method?

```Python
value = value.encode("utf-16")
```

Reply:
I think so! I can try that